### PR TITLE
New functional tests

### PR DESCRIPTION
**Pull request by buixor**
- install crowdsec from release (`--bininstall`)
 - start service with systemctl
 - inject logs into another instance in time-machine
 - check (with `cscli decisions list` + filters) expected results
 - delete existing bans
 - add a new file to acquis.yaml
 - restart service
 - inject same logs but in tail mode
 - check results again

### CODE REVIEW

File: .github/workflows/ci_functests-install.yml [hunk 1]
```diff
@@ -2,9 +2,13 @@ name: Hub-CI
...
   push:
-    branches: [ wip_lapi_xxx_no ]
+    branches:
+      - wip_lapi
+      - master
   pull_request:
-    branches: [ wip_lapi_xxx_no ]
+    branches:
+      - wip_lapi
+      - master
```
Comment (registergoofy) [inline .github/workflows/ci_functests-install.yml:11]: This'll have to be changed in the final release.

File: wizard.sh [hunk 2]
```diff
@@ -433,13 +433,6 @@ main() {
         fi
         log_info "installing crowdsec"
         install_crowdsec
-        # lapi register
-        MACHINE_ID=""
-        if [[ ${DOCKER_MODE} == "false" ]]; then
-            ${CSCLI_BIN_INSTALLED} machines add --force "$(cat /etc/machine-id)" --password "$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)" -f "${CROWDSEC_CONFIG_PATH}/${CLIENT_SECRETS}"
-        fi
-            
-        log_info "Crowdsec LAPI registered"
```
Comment (registergoofy) [inline wizard.sh:442]: Why get rid of this ?

**Review by registergoofy — COMMENTED** [review]
first comment is cosmetics
second comment is more a question than a comment

File: wizard.sh (hunk 2 above)
Comment (buixor) [reply wizard.sh:442]: The official one is that: `--bininstall` doesn't have to register to LAPI. The less official one is that `$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)` caused the docker image of CI to hang

File: .github/workflows/ci_functests-install.yml (hunk 1 above)
Comment (buixor) [reply .github/workflows/ci_functests-install.yml:11]: I guess we can just remove `wip_lapi` ? or you see something else?

File: wizard.sh (hunk 2 above)
Comment (registergoofy) [reply wizard.sh:442]: ok then

File: .github/workflows/ci_functests-install.yml (hunk 1 above)
Comment (registergoofy) [reply .github/workflows/ci_functests-install.yml:11]: fair enough

**Review by registergoofy — APPROVED** [review]
lgtm

**Pull request merged**